_topics/en/time-warp.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Time warp
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Mining
+  - Security Problems
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: bip-cleanup, describing a solution to the time warp attack
+      link: https://github.com/TheBlueMatt/bips/blob/cleanup-softfork/bip-XXXX.mediawiki
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: Consensus cleanup soft fork propsed to eliminate time warp attack
+    url: /en/newsletters/2019/03/05/#fix-the-time-warp-attack
+
+  - title: Background on time warp attack for consensus cleanup soft fork
+    url: /en/newsletters/2019/03/05/#the-time-warp-attack
+
+  - title: "Forward blocks proposal to use time warp to technically avoid hard forks"
+    url: /en/newsletters/2018/10/09/#forward-blocks-on-chain-capacity-increases-without-a-hard-fork
+
+  - title: Discussion from Scaling Bitcoin V conference about fixing the time warp bug
+    url: /en/newsletters/2018/10/09/#fixing-the-time-warp-bug
+
+  - title: Request for soft fork solutions to the time warp attack
+    url: /en/newsletters/2018/08/28/#requests-for-soft-fork-solutions-to-the-time-warp-attack
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Consensus cleanup
+    link: topic consensus cleanup
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Time warp** is an attack that allows a majority of miners to produce
+  up to one block per second indefinitely.  The attack could be used to
+  steal the remaining block subsidy or to perform a DoS attack on nodes
+  and clients.
+
+---
+
+Bitcoin blocks contain an nTime field in their headers.  We want this to
```

Comment:
Maybe "The value of this field is supposed to", or "The value of this field is typically set to" (or the like), instead of "We want this to"?